eth_worker/eth_manager/processor.py

```diff
@@ -200,7 +251,8 @@ def check_transaction_hash(self, tx_hash):
         tx_receipt = self.w3.eth.getTransactionReceipt(tx_hash)
 
         def print_and_return(return_dict):
```

Comment:
`print_and_return` only returns 😒

Reply:
u saw the comment below though yeah? Just commented it out for now incase we want it later